### ISSUE TITLE
Example packaged notebooks load with error "not found"

### ISSUE THREAD

**Issue by dyerrington** [andypetrella/spark-notebook#389]
![](http://snag.gy/3ZcnP.jpg)

Console messages:
```
[info] application - View notebook 'sql/Using Spark SQL.snb'
[debug] application - content: sql/Using Spark SQL.snb
[debug] application - getNotebook: name is 'sql/Using Spark SQL', path is 'sql/Using Spark SQL.snb' and format is 'json'
[info] application - getNotebook at path sql/Using Spark SQL.snb
[info] application - Loading notebook at path sql/Using Spark SQL.snb
```
[Startup console messages](http://pastebin.com/tzswJvDy)

It would be cool to see these examples. Is it possible I'm overlooking something obvious?

**Comment by dyerrington** [comment]
For anyone else interested, I found that running the notebook from any other directory, other than what's relative to the ./notebooks directory, produced this "file not found".  I saw nothing in the console.  Not sure if this is expected but hopefully this helps the project, and other people who run into this from search engines.

**Comment by andypetrella** [comment]
Right! it's indeed a problem that should have gone with a simplistic heuristic to find it around... but looks like the classpath management is against the notebook :-)

Thanks for creating the issue, this has to be fixed!

**Comment by shadjiiski** [comment]
That's really strange. I am keeping all of my notebooks in a subfolder of *notebooks* and have never experienced this problem. Also, I have opened the *sql/Using Spark SQL.snb* with no trouble at all

**Comment by markon** [comment]
I had this problem solved by launching the notebook from the root directory, as explained in the readme: 

> Change to the root directory of the expanded distribution.
> Execute the command bin/spark-notebook (*NIX) or bin\spark-notebook (Windows).

**Comment by AhmedKamal** [comment]
@markon  It worked for me too following your tip :+1:  Thanks !

**Comment by NKozlov** [comment]
@markon thx!
